tests/e2e/user-acceptance.spec.ts

```
@@ -0,0 +1,103 @@
+    // Input validation (login with invalid email)
+    await page.goto('/login')
+    await page.fill('input[type="email"]', 'invalid-email')
+    await page.fill('input[type="password"]', 'password123')
+    await page.click('button[type="submit"]')
+    await expectToastNotification(page, /invalid|error|email/i, 'error')
+
+    // 6. Accessibility & Performance (MCP Integration)
+    // Accessibility audit (if MCP is available)
+    if (typeof window !== 'undefined' && 'mcp_browser_tools_runAccessibilityAudit' in window) {
+      const accessibilityResults = await page.evaluate(async () => {
+        // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)